resources/getDataFromStack.js

```diff
@@ -0,0 +1,33 @@
+
+let bigJson = [];
```

Comment:
Let's move all requests to StackOverflow to the server, and pass only required data to the client.